troubadix/helper/patterns.py

```diff
@@ -47,6 +47,7 @@ def _get_tag_pattern(
 ) -> re.Pattern:
     """
     The returned pattern catches all `script_tags(name="", value="");`
```

Comment:
This would better indicate how the `name`and `value` parameters are used.
`The returned pattern catches all `script_tags(name="{name}", value="{value}");`